File: OAIP/LR-1-2/3.2.py
a = float(input("Введите первое число: "))
b = float(input("Введите второе число: "))
c = float(input("Введите третье число: "))
count = 0
if a > 0:
    count += 1
if b > 0:
    count += 1
if c > 0:
    count += 1
print("Количество положительных чисел: ", count)

File: OAIP/LR-1-2/3.2.pas
program polozhitelnye;

var
  a, b, c: real;
  count: integer;

begin
  writeln('Введите три числа:');
  readln(a, b, c);
  count := 0;
  if a > 0 then
    count := count + 1;
  if b > 0 then
    count := count + 1;
  if c > 0 then
    count := count + 1;
  writeln('Количество положительных чисел: ', count);
end.